templates/solutions/open-source-security/cyber-resilience-act/index.html

```diff
@@ -0,0 +1,591 @@
+{% extends "templates/base.html" %}
+
+{% from "_macros/vf_tiered-list.jinja" import vf_tiered_list %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/13VaiP6GGkGB4mE01bIqJ3cg8MfQXulNs9m3cmYb20HQ
+{% endblock meta_copydoc %}
+
+{% block title %}Understand the Cyber Resilience Act (CRA) and its requirements{% endblock %}
+
+{% block meta_description %}
+  Discover what you need to know about the EU Cyber Resilience Act requirements, and get clear recommendations for compliance. Understand how Canonical can help.
+{% endblock %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block outer_content %}
+
+  {% block content %}
+
+    <section class="p-section--hero">
```

Comment:
We are using hero patterns from Vanilla for hero sections, can you update it to use the pattern? Link to the doc [here](https://vanillaframework.io/docs/patterns/hero)

Reply:
Doesnot follow exactly any of the hero patterns as all of them require an image but this hero doesnot have an image